apache/vhosts/minimal.tmpl

```diff
@@ -13,6 +13,8 @@
     'LogLevel': site.get('LogLevel', 'warn'),
     'ErrorLog': site.get('ErrorLog', '{0}/{1}-error.log'.format(map.logdir, sitename)),
     'CustomLog': site.get('CustomLog', '{0}/{1}-access.log'.format(map.logdir, sitename)),
+    'LogFormat': site.get('LogFormat', '"%h %l %u %t \\\"%r\\\" %>s"'),
```

Comment:
As per [current Apache documentation](http://httpd.apache.org/docs/current/mod/mod_log_config.html#logformat), the default `LogFormat` is `"%h %l %u %t \"%r\" %>s %b"`. You seem to have missed the last `%b`. Mind you add it, so log format doesn't change when this is added in new runs?